reoptjl/models.py

```diff
@@ -296,7 +296,7 @@ class SOLVERS(models.TextChoices):
 
     solver_name = models.TextField(
         blank=True,
-        default=SOLVERS.XPRESS,
+        default=SOLVERS.HIGHS,
```

Comment:
This might not be critical for this PR, but if you use an empty string (below) for the solver_name input, it does not automatically select HiGHS and the error under "messages" is just `Unexpected Error...`

`post_1["Settings"]["solver_name"] = '' or ""`

Reply:
Ok, yeah I'll punt on this for now. I don't think the user would enter an empty string accidentally in most cases.